go.mod

```
@@ -2,7 +2,11 @@ module github.com/ONSdigital/dp-net/v2
 
 go 1.19
 
-retract v2.7.0
+retract (
+	v2.7.0
+	v2.7.1 // rethink application of timeout to standard  http server
```

This looks like a TODO:
```suggestion
	v2.7.1 //TODO rethink application of timeout to standard  http server
```